tests/e2e/regenerate_traces.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+go test -timeout 30s -run ^TestWriteExamples$ github.com/cosmos/interchain-security/v3/tests/e2e
```

Comment:
Seems like this'd be more appropriate as a make target

Reply:
True, good call